.github/workflows/security-scans.yaml

```diff
@@ -0,0 +1,29 @@
+name: 'Security scans to zotregistry.io'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: read
+  packages: write
+
+jobs:
+  zap_scan:
+    runs-on: ubuntu-latest
+    name: Scan the webapplication
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: master
+      - name: ZAP Scan (zotregistry.io)
+        uses: zaproxy/action-baseline@v0.7.0
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          docker_name: 'owasp/zap2docker-stable'
+          target: 'https://zotregistry.io'
+          rules_file_name: '.zap/rules.tsv'
```

Comment:
I believe this should be included in the commit as well. Unless we use the defaults, in which case I don't think this config option is needed here.

Reply:
added an empty file (for now). we can add rules there as we learn.